### PR TITLE
Fixed createElement, hasAttribute issues

### PR DESCRIPTION
**Pull request by denisdulici**
Refs https://github.com/dompdf/dompdf/issues/1551, https://github.com/dompdf/dompdf/issues/1563

### CODE REVIEW

File: src/Css/Stylesheet.php [hunk 1]
```diff
@@ -965,6 +966,11 @@ function apply_styles(FrameTree $tree)
 
                 /** @var \DOMElement $node */
                 foreach ($nodes as $node) {
+                    // Expect $node to be an instance of DOMElement class
+                    if (!$node instanceof DOMElement) {
```
Comment (bsweeney) [inline src/Css/Stylesheet.php]: Shouldn't we wrap the statement in parenthesis (`if (!($node instanceof DOMElement)) {...}`?

Comment (denisdulici) [reply src/Css/Stylesheet.php]: Not really, as far as I know. It works fine for me and @akaunting users.

Comment (bsweeney) [reply src/Css/Stylesheet.php]: You're right, in the [operator precedence documentation](http://php.net/manual/en/language.operators.precedence.php) `instanceof` comes before the negation operator. I guess not having the forced precedence makes me a bit uncomfortable. However, as you stated, not required.

**Comment by bsweeney** [comment]
After looking at why this error was happening I noticed we were already performing this check in the non-generated content style logic block, but with a slightly different condition. I modified the code to use that condition so that the two match up (a76c706).

**Pull request merged**